translations/strings.yaml

```diff
@@ -73,6 +73,30 @@
 #
 
 serversView:
+  recommendedCardTitle:
+    value: Why are these locations recommended?
+    comment: Title of an expandable info card on the recommended servers list.
+  recommendedCardBody:
+    value: These locations are sorted by expected performance.
+    comment: Body text of an expandable info card on the recommended servers list.
+  recommendedTextStatus:
+    value: "Last updated %1 ago."
+    comment: Time passed since the last update to the recommended server list. %1 is the time value and unit.
+  recommendedTextStatusWithInfo:
+    value: "Last updated %1 ago. To update this list please first disconnect from the VPN."
+    comment:
+      - Time passed since the last update to the recommended server list and additional info.
+      - "%1 is the time value and unit since the list was updated."
```

Comment:
For this and the string before we should specify how the time value and unit are generated, i.e. if they're localized and how they will look like (I see a `// TODO: Replace string arg with the actual formatted date` comment in the code). How are these going to be generated?

Reply:
I was thinking that generating the time value and unit like we do  in [AddonMessage::dateInternal](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/src/addons/addonmessage.cpp#L165) would work, but thinking about it again this might be subject to change. I’d rather not introduce these strings with this PR and do it in a follow up.